proto/order.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package order;
+
+option java_package = "com.risingwave.proto";
+option optimize_for = SPEED;
+
+enum PbDirection {
```

Comment:
so a new naming method here, +1 because I have been tired to writing `use XXX as ProstXXX`.

Reply:
@BugenZhao and I have discussed about a new method to avoid writing `use Xxx as ProstXxx` again and again, which will be nicer. For now, naming these types `PbXxx` really comfort my coding experience😄

Reply:
Yes. With the proc-macro generated alias, we can avoid manually prefixing them with `Pb` here and directly using it.